### PR TITLE
Bugfix/colored admin table

### PR DESCRIPTION
**Pull request by katia9261**

### CODE REVIEW

File: src/app/ubs/ubs-admin/components/ubs-admin-table/table-cell-readonly/table-cell-readonly.component.scss [hunk 1]
```diff
@@ -13,6 +15,18 @@ div {
   padding: 0 20px;
 }
 
+.un-paid {
+  background-color: var(--primary-white) !important;
+}
+
+.paid {
+  background-color: #ff0 !important;
```
Comment (hnativlyubomyr) [inline src/app/ubs/ubs-admin/components/ubs-admin-table/table-cell-readonly/table-cell-readonly.component.scss]: add color variable here

File: src/app/ubs/ubs-admin/components/ubs-admin-table/table-cell-readonly/table-cell-readonly.component.scss [hunk 2]
```diff
@@ -13,6 +15,18 @@ div {
...
+}
+
+.half-paid {
+  background-color: #fff9d2 !important;
```
Comment (hnativlyubomyr) [inline src/app/ubs/ubs-admin/components/ubs-admin-table/table-cell-readonly/table-cell-readonly.component.scss]: the same

File: src/app/ubs/ubs-admin/components/ubs-admin-table/ubs-admin-table.component.ts [hunk 3]
```diff
@@ -353,6 +353,23 @@ export class UbsAdminTableComponent implements OnInit, AfterViewChecked, OnDestr
     return `${this.selection.isSelected(row) ? 'deselect' : 'select'} row ${row.id + 1}`;
   }
 
+  isStatus(row): string {
+    switch (row.orderStatus) {
+      case OrderStatus.CANCELED:
```
Comment (hnativlyubomyr) [inline src/app/ubs/ubs-admin/components/ubs-admin-table/ubs-admin-table.component.ts:358]: you can group cases

case OrderStatus.CANCELED:
case OrderStatus.BROUGHT_IT_HIMSELF:
        return '#CCCCCC';

File: src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-location-pop-up/ubs-admin-tariffs-location-pop-up.component.ts [hunk 4]
```diff
@@ -417,8 +417,10 @@ export class UbsAdminTariffsLocationPopUpComponent implements OnInit, AfterViewC
     trigger.openPanel();
   }
 
-  private checkCityExist(item: string, array: Array<string>): boolean {
-    const newCityName = item.toLowerCase();
+  private checkCityExist(item, array: Array<string>): boolean {
+    console.log('item', item, 'array', array);
```
Comment (hnativlyubomyr) [inline src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-location-pop-up/ubs-admin-tariffs-location-pop-up.component.ts]: remove console.log)

**Review by hnativlyubomyr — APPROVED** [review]

File: src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-location-pop-up/ubs-admin-tariffs-location-pop-up.component.ts (hunk 4 above)
Comment (katia9261) [reply src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-location-pop-up/ubs-admin-tariffs-location-pop-up.component.ts]: thanks, deleted it

File: src/app/ubs/ubs-admin/components/ubs-admin-table/ubs-admin-table.component.ts (hunk 3 above)
Comment (katia9261) [reply src/app/ubs/ubs-admin/components/ubs-admin-table/ubs-admin-table.component.ts:358]: thanks, added this change

File: src/app/ubs/ubs-admin/components/ubs-admin-table/table-cell-readonly/table-cell-readonly.component.scss (hunk 2 above)
Comment (katia9261) [reply src/app/ubs/ubs-admin/components/ubs-admin-table/table-cell-readonly/table-cell-readonly.component.scss]: added

File: src/app/ubs/ubs-admin/components/ubs-admin-table/table-cell-readonly/table-cell-readonly.component.scss (hunk 1 above)
Comment (katia9261) [reply src/app/ubs/ubs-admin/components/ubs-admin-table/table-cell-readonly/table-cell-readonly.component.scss]: I thought that creating a constant for a color that is not included in our design is pointless)).
Added changes and at the same time made constants for color numbers in the .ts file

**Review by NadiiaStoiko — APPROVED** [review]

**Review by ruslan-teh — APPROVED** [review]

**Review by Tadei-Mykola — APPROVED** [review]

**Review by mdashko — APPROVED** [review]

**Review by OksanaFilipchuk — APPROVED** [review]

**Pull request merged**